### DISCUSSION TITLE
V2 vs. V3

### DISCUSSION THREAD

**Discussion by MariamMahdy** [DrCoffey/DeepSqueak#166] [Q&A]
Hello,
I ran the same audio file on both versions of Deepsqueak (V.2.6 and V.3.2), expecting that V3 would be better. However, V2 showed 128 calls, 90% of them are real calls and around 10% noise. V3 showed 1698 calls! and only 10% are actual calls. The overall quality of the spectogram in V2 was also way better. So, I am kinda confused, is there something extra that i should do in V3? 
Noting that i never applied a denoising network in V2 and never trained the network in anyway.

I would really appreciate your help :)

**Comment by DrCoffey** [comment]
Hey @MariamMahdy, the new rat detection network is very sensitive and can pick up noise more easily. Do you have some consistent noise band in the audio file? You can try the mouse call network, it is often more robust to noise. I will try to release some alternative networks in the future that are more robust to noisy frequency bands. If you share the audio file I can be more helpful.

As for the spectrogram quality, you can change all of the settings with the display buttons on the bottom right. Choose "amplitude" to make it look more like v2.

**Reply by MariamMahdy** [reply]
Thank you so much for your reply!
I am actually using a mouse network since the recording (and my entire project) is vocalization in mice. However, the recorder that we are using is different (not an avisoft mic but an audiomoth) so it has more noise.

https://drive.google.com/drive/folders/1JnKsBAefwhkiBDzWkz3zHqq8Hgbf3mE-?usp=sharing

This link has a sample audio from the recording (there's 13 more files) 
I guess the main problem is that many noise fragments are counted at calls.

**Comment by DrCoffey** [comment]
Hey Mariam,

Sorry, I just got around to this and it appears the folder is empty.

Thanks,
Kevin
